### ISSUE TITLE
darwin: openexr build failure blocks nixpkgs-unstable

### ISSUE THREAD

**Issue by FRidh** [NixOS/nixpkgs#86438]
**Describe the bug**
darwin: openxr build failure blocks nixpkgs-unstable

**To Reproduce**
https://hydra.nixos.org/build/117777551

**Notify maintainers**
cc @NixOS/darwin-maintainers 

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
- `openexr`
# a list of nixos modules affected by the problem
module:
```

**Comment by flokli** [comment]
This is tracked in https://github.com/NixOS/nixpkgs/pull/86329.

**Comment by flokli** [comment]
I merged in #86329 to unblock channels.
